File: app/src/main/java/net/mmho/photomap2/MapSuggestionProvider.java
package net.mmho.photomap2;

import android.content.SearchRecentSuggestionsProvider;

public class MapSuggestionProvider extends SearchRecentSuggestionsProvider{
    public final static String AUTHORITY = "net.mmho.MapSuggestionProvider";
    public final static int MODE = DATABASE_MODE_QUERIES;

    public MapSuggestionProvider(){
        setupSuggestions(AUTHORITY,MODE);
    }
}

File: app/src/main/kotlin/net/mmho/photomap2/MapSuggestionProvider.kt
package net.mmho.photomap2

import android.content.SearchRecentSuggestionsProvider

class MapSuggestionProvider : SearchRecentSuggestionsProvider() {

    init {
        setupSuggestions(AUTHORITY, MODE)
    }

    companion object {
        const val AUTHORITY = "net.mmho.MapSuggestionProvider"
        const val MODE = SearchRecentSuggestionsProvider.DATABASE_MODE_QUERIES
    }
}
